src/offset.js

```diff
@@ -4,17 +4,17 @@ var oldOffset = jQuery.fn.offset;
 jQuery.fn.offset = function() {
 	var docElem,
 		elem = this[ 0 ],
-		origin = { top: 0, left: 0 };
+		bogus = { top: 0, left: 0 };
 
 	if ( !elem || !elem.nodeType ) {
 		migrateWarn( "jQuery.fn.offset() requires a valid DOM element" );
-		return origin;
```

Comment:
We only want to return `undefined` for the `!elem` case. Some code including older older versions of jQuery UI would pass `window` and expect the top/left of 0. https://github.com/jquery/jquery/issues/3249

Reply:
Does it expect `{left:0,top:0}` for `window`? Why both `1.12.4` and `2.2.4` return `undefined` as I tested?

Reply:
It sure does, thanks for catching that! I don't have time to look at it right now but I must have been thinking of some other edge case with `.offset()`? Looks like that ticket is specifically about calling a method on `window`, not returning the bogus value.

Reply:
@dmethvin Since the issue was about a breaking change in jQuery 3.0.0, it seems we shouldn't care about it here as Migrate 3.x is meant to aid with upgrading to the future jQuery 4.0. Am I missing anything?